tests/src/test_layer_stack.cpp

```
@@ -0,0 +1,154 @@
+        bool updateCalled = false;
+        bool drawCalled = false;
+        uint32_t lastTicks = 0;
+        bool returnValue = false;
+
+        bool OnEvent(Event const& event) override { (void)event; onEventCalled = true; return returnValue; }
+        void Update(uint32_t ticks) override { updateCalled = true; lastTicks = ticks; }
+        void Draw() override { drawCalled = true; }
+    };
+
+    struct TestEvent : Event {
+        TestEvent() : Event(100) {}
```

Clang (14.0.6)

[error] 100 is a magic number; consider replacing it with a named constant

(cppcoreguidelines-avoid-magic-numbers,-warnings-as-errors)